test/e2e/test/elasticsearch/checks_es.go

```diff
@@ -28,10 +28,10 @@ func (b Builder) CheckStackTestSteps(k *test.K8sClient) test.StepList {
 		k:  k,
 	}
 	return test.StepList{
-		e.CheckESReachable(),
+		e.CheckESNodesTopology(b.Elasticsearch),
 		e.CheckESVersion(b.Elasticsearch),
+		e.CheckESReachable(),
```

Comment:
We can probably get rid of `CheckESReachable `, which is now implied by `CheckESNodesTopology `. It's just checking cluster health, which is also done by `CheckESHealthGreen ` right after.